chroot-script

```diff
@@ -691,14 +743,18 @@ grub_install() {
      grub_device="${GRUB}"
   fi
 
-  echo "Setting ${GRUB_PACKAGE} debconf configuration for install device to $GRUB"
-  echo "${GRUB_PACKAGE} ${GRUB_PACKAGE}/install_devices multiselect ${grub_device}" | debconf-set-selections
-
-  if ! dpkg --list "${GRUB_PACKAGE}" 2>/dev/null | grep -q '^ii' ; then
-    echo "Notice: grub option set but no ${GRUB_PACKAGE} package, installing it therefore."
-    DEBIAN_FRONTEND=$DEBIAN_FRONTEND $APTINSTALL "${GRUB_PACKAGE}"
+  echo "Setting grub debconf configuration for install device to $GRUB"
+  if [ -n "$MAIN_GRUB_PACKAGE" ]; then
+    echo "${MAIN_GRUB_PACKAGE} ${MAIN_GRUB_PACKAGE}/install_devices multiselect ${grub_device}" | debconf-set-selections
   fi
 
+  for GRUB_PACKAGE in "$MAIN_GRUB_PACKAGE" "${GRUB_SECBOOT_PACKAGES[@]}"; do
+    if ! dpkg --list "${GRUB_PACKAGE}" 2>/dev/null | grep -q '^ii' ; then
```

Comment:
why not skip the check and just install the packages?

Reply:
The only thing that would be lost is the messaging about the packages being installed, which might be handy for debugging purposes, maybe. I think the only reason I did this the way I did was to mimic the existing behavior as much as possible. If it would be better to optimize that to a simple apt install, I'm happy to do that.

Reply:
Tested, works when building a Bookworm image with hybrid boot support on a BIOS system.